### PR TITLE
Add support for passing new Spring Boot JarLauncher class

### PR DESCRIPTION
**Pull request by hjellek**
Attempt at fixing https://github.com/salesforce/rules_spring/issues/177

Works in our project, and has backward compatability.

Open to suggestions for improvements!

### CODE REVIEW

**Comment by hjellek** [comment]
I have signed the CLA now

File: springboot/springboot.bzl [hunk 1]
```diff
@@ -385,6 +385,7 @@ def springboot(
         name,
         java_library,
         boot_app_class,
+        spring_boot_launcher_class = "org.springframework.boot.loader.JarLauncher",
```
Comment (plaird) [inline springboot/springboot.bzl:388]: nitpick: for consistency, can you rename attr to boot_launcher_class

**Review by plaird — CHANGES_REQUESTED** [review]
One small nitpick, otherwise looks good. Thanks!

File: springboot/springboot.bzl (hunk 1 above)
Comment (hjellek) [reply springboot/springboot.bzl]: Of course

**Review by plaird — APPROVED** [review]

**Comment by plaird** [comment]
I am preparing release 2.3.1 with this change.

**Pull request merged**